### DISCUSSION TITLE
the plugin does not work Clearfy Pro

### DISCUSSION THREAD

**Discussion by Malik741111** [community/community#6051] [Codespaces]
the Clearfy Pro plugin does not work
I can't install the plugin on my site in any way [https://oblok.ru](https://oblok.ru) What should I do? I download the program but writes that the program cannot be activated

**Comment by bamurtaugh** [comment]
Thanks @Malik741111 - could you please provide some more info? Are you using GitHub Codespaces or the github.dev web-based editor? And could you please provide a link to the Clearfy Pro extension - I cannot seem to find it upon a search of the marketplace.
